src/test/kotlin/lotto/domain/MoneyTest.kt

```diff
@@ -60,6 +60,6 @@ internal class MoneyTest {
     @Test
     fun `money 객체를 bigDecimal로 형변환 할 수 있다`() {
         val actual = Money(300.toBigInteger()).toBigDecimal()
-        assertThat(actual as? BigDecimal).isNotNull()
+        assertThat(actual as? BigDecimal).isInstanceOf(BigDecimal::class.java)
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        assertThat(actual).isInstanceOf(BigDecimal::class.java)
```